utils/trackerbot.py

```diff
@@ -197,11 +197,14 @@ def templates_to_test(api, limit=1):
 
     """
     templates = []
-    for pt in api.untestedtemplate.get(limit=limit, tested=False).get('objects', []):
+    for pt in api.untestedtemplate.get(
+            limit=limit, tested=False, provider__type=request_type).get(
```

Comment:
If we pass ```None``` to the API call here, does it still work in the backwards compatible fashion?